src/components/Header/Header.tsx

```diff
@@ -21,11 +23,17 @@ const Header = () => {
       case '/leverage':
         setCurrentPage('your-position');
         break;
-      case '/redeem':
-        setCurrentPage('redeem');
+      case '/savings':
+        setCurrentPage('savings');
+        break;
+      case '/bridge':
+        setCurrentPage('bridge');
+<<<<<<< HEAD
```

Comment:
Artifacts from merge conflict left in code. Try to run `npm run typecheck` to make sure all types and syntax is correct.

Reply:
resolved